protofiles/grpc.proto

```diff
@@ -4,33 +4,49 @@ package protofiles;
 import "google/api/annotations.proto";
 import "messages.proto";
 
```

Comment:
should we have the Java, C#, etc options in this file too?

Reply:
We are importing messages.proto, So I am not sure if it will be redundant.

Reply:
I have a feeling you will need it still